dashboard/test/factories/factories.rb

```diff
@@ -328,6 +328,17 @@
       end
     end
 
+    trait :multi_auth_migrated do
+      after(:create) do |user|
+        user.update_attributes(
+          provider: 'migrated',
+          uid: '',
+          email: '',
+          hashed_email: ''
```

Comment:
This makes a lot of sense for tests.  Do you think we need to do this for migrated users before we remove those columns, just so we detect problems earlier?  Or could we do this manually for some internal users and be okay?

Reply:
I'm not sure, actually - it might make sense to wipe stuff out for at least more than just internal users before deleting the columns. If that deletion deployment goes badly and we discover new stuff, it'll be a *lot* harder to undo that.